lnpeer/peer.go

```diff
@@ -0,0 +1,26 @@
+package lnpeer
+
+import (
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/roasbeef/btcd/btcec"
+	"github.com/roasbeef/btcd/wire"
+)
+
+// Peer is an interface which represents the remote lightning node inside our
+// system.
+type Peer interface {
+	// SendMessage sends a variadic number of message to remote peer. The
+	// first argument denotes if the method should block until the message
+	// has been sent to the remote peer.
+	SendMessage(sync bool, msg ...lnwire.Message) error
+
+	// WipeChannel removes the channel uniquely identified by its channel
+	// point from all indexes associated with the peer.
+	WipeChannel(*wire.OutPoint) error
```

Comment:
Not a blocker, but we may want to reconsider the usage of this method in the codebase in general.

Reply:
for sure, looks like it's really only used in the htlcswitch. can def be moved out of the toplevel interface, haven't really thought about alternatives to this atm